hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -242,6 +242,14 @@ public boolean isActive() {
                 && daggerApp.grpcServerManager().isRunning();
     }
 
+    /**
+     * Indicates whether this node is FROZEN.
+     * @return True if the platform is frozen
+     */
+    public boolean isFrozen() {
+        return platformStatus == PlatformStatus.FREEZE_COMPLETE;
+    }
```

Comment:
Added so that the node control code in the test system can know if the node is frozen. Only applies to an in-process "Alice", which we are not using in this PR. So technically, this method wasn't needed in this PR.